### ISSUE TITLE
deploying

### ISSUE THREAD

**Issue by mikebutts** [artofbryce/jammming#1]
https://github.com/artofbryce/jammming/blob/bb686a1be688884dabb213035da302feff95092c/src/util/Spotify.js#L5

Have you tried deploying this? if not give it a try. 

https://medium.freecodecamp.org/surge-vs-github-pages-deploying-a-create-react-app-project-c0ecbf317089

**Comment by artofbryce** [comment]
Yes, done! 🙂Had some issues with surge before but now it's working: http://brycejammming.surge.sh/
